File: test/fsharpTests/CharTest/isLetterTest.fs
testCase "isLetter('A')" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Char.isLetter 'A') "error"
testCase "isLetter('7')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isLetter '7') "error"
testCase "isLetter(' ')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isLetter ' ') "error"
testCase "isLetter('\n')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isLetter '\n') "error"
testCase "isLetter('\001')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isLetter '\001') "error"
testCase "isLetter('\236')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isLetter '\236') "error"

File: test/ocamlTests/CharTest/isLetterTest.ml
test "isLetter('A')" (fun () -> expect (Char.isLetter 'A') |> toEqual Eq.bool true) ; 
test "isLetter('7')" (fun () -> expect (Char.isLetter '7') |> toEqual Eq.bool false) ; 
test "isLetter(' ')" (fun () -> expect (Char.isLetter ' ') |> toEqual Eq.bool false) ; 
test "isLetter('\n')" (fun () -> expect (Char.isLetter '\n') |> toEqual Eq.bool false) ; 
test "isLetter('\001')" (fun () -> expect (Char.isLetter '\001') |> toEqual Eq.bool false) ; 
test "isLetter('\236')" (fun () -> expect (Char.isLetter '\236') |> toEqual Eq.bool false) ; 
